inbc-program/inbc/command/command.py

```diff
@@ -129,7 +129,7 @@ def invoke_update(self, args: Any) -> None:
 
         @param args: arguments passed to command-line tool.
         """
-        super()._send_manifest(args, RESTART_CHANNEL)
+        super()._send_manifest(args, RESTART_CMD_CHANNEL)
 
     def search_response(self, payload: Any) -> None:
         """Search for keywords in response message
```

Comment:
**<h2>$\color{red}Defects: Security Vulnerability$</h2>**
**Suggestion:** The search_response method in the QueryCommand class prints the payload directly. This could lead to sensitive information being logged. Consider sanitizing the payload or using a more secure logging method.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
def search_host_response(self, payload: Any) -> None:
    sanitized_payload = self.sanitize_payload(payload)
    logger.info(sanitized_payload)

def sanitize_payload(self, payload: Any) -> str:
    # Implement payload sanitization logic here
    return sanitized_payload
```